src/shared/components/palette/money-gbp/write-money-gbp-field.component.spec.ts

```diff
@@ -52,8 +125,8 @@ describe('WriteMoneyGbpFieldComponent', () => {
     fixture = TestBed.createComponent(WriteMoneyGbpFieldComponent);
     component = fixture.componentInstance;
 
+    component.registerControl = REGISTER_CONTROL;
     component.caseField = CASE_FIELD;
-    component.formGroup = FORM_GROUP;
```

Comment:
Same question as above. This works elsewhere so why has it been replaced with an old mechanism?

Reply:
Same answer as before. Just copied the changes from the PR.